v23.1/take-and-restore-encrypted-backups.md

```diff
@@ -60,6 +61,52 @@ The AWS URI **requires** the following:
 
 See AWS's [KMS keys](https://docs.aws.amazon.com/kms/latest/developerguide/create-keys.html) documentation for guidance on creating an AWS KMS key.
 
+#### Azure Key Vault URI format
+
+{% include_cached new-in.html version="v23.1" %} The Azure Key Vault URI must use one of two formats:
+
+- Explicit authentication using the `AUTH=specified` parameter (or omitting this as per the example) with the tenant ID, client ID, client secret, and key vault name parameters:
```

Comment:
```suggestion
- Explicit authentication using the `AUTH=specified` parameter (or omitting this, as it is the default option) with the tenant ID, client ID, client secret, and key vault name parameters:
```

Potentially clearer, if true (i.e. if I understand the implication in your draft correctly)?

Reply:
Yes, I like it! Done!